src/js/models/panel_model.js

```diff
@@ -708,5 +708,38 @@
             }
         },
 
+        createLabelsFromTags(options) {
+            // Loads Tags for selected images and creates labels
+            var image_ids = this.map(function(s){return s.get('imageId')})
+            image_ids = "image=" + image_ids.join("&image=");
+            // TODO: Use /api/ when annotations is supported
+            var url = WEBINDEX_URL + "api/annotations/?type=tag&limit=1000&" + image_ids;
```

Comment:
maybe offering the option later on to change the limit.
this will also depend on the number of images

Reply:
This limit is just a very liberal cut-off to make sure we don't accidentally load millions of annotations, but that scenario is very edge-case.